### PR TITLE
Update support libraries to 26.0.0.

### PR DESCRIPTION
**Pull request by NightlyNexus**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -106,7 +106,9 @@ subprojects {
     resolutionStrategy {
       eachDependency { details ->
         // Force all of the primary support libraries to use the same version.
-        if (details.requested.group == 'com.android.support') {
+        if (details.requested.group == 'com.android.support'
+            && details.requested.name != 'multidex'
+            && details.requested.name != 'multidex-instrumentation') {
```
Comment (JakeWharton) [inline build.gradle]: Do we really need this?

Comment (NightlyNexus) [inline build.gradle]: Oops, I forgot to comment in the PR...

The design library (com.android.support:design:26.0.0) brings in com.android.support:multidex:1.0.1

Comment (JakeWharton) [inline build.gradle]: Uh, why? And do we depend on another version or something?

Comment (NightlyNexus) [inline build.gradle]: https://issuetracker.google.com/issues/63999442
I could `exclude module: 'multidex'` if you'd like, or we could wait for the fix.

Comment (NightlyNexus) [inline build.gradle]: >do we depend on another version or something?

no, there's just no 26.0.0 of the multidex library, so we can't force that version.

Comment (JakeWharton) [inline build.gradle]: Oh yeah missed the `!=`. This transitive dep is bad news. Good thing we all have pull request commenting robots now, right?!??

Comment (NightlyNexus) [inline build.gradle]: hehe. i'm actually gonna file a ticket at my company citing this example in the hopes of my one day building one.

File: build.gradle [hunk 2]
```diff
@@ -106,7 +106,9 @@ subprojects {
     resolutionStrategy {
       eachDependency { details ->
         // Force all of the primary support libraries to use the same version.
-        if (details.requested.group == 'com.android.support') {
+        if (details.requested.group == 'com.android.support'
+            && details.requested.name != 'multidex'
```
Comment (tasomaniac) [inline build.gradle]: You may also do above just before `resolutionStrategy`
```
configurations {
    compile.exclude module: 'multidex'
}
```

File: build.gradle (hunk 1 above)
Comment (JakeWharton) [inline build.gradle]: Let's do a local exclude with a TODO comment to remove once 26.0.1 is released.

Comment (JakeWharton) [inline build.gradle]: Actually this is fine.

**Pull request merged**